README.md

```diff
@@ -1,45 +1,57 @@
-# WAZP - **W**asp **A**nimal-tracking **Z**oo project with **P**ose estimation
+[![License](https://img.shields.io/badge/License-BSD_3--Clause-orange.svg)](https://opensource.org/licenses/BSD-3-Clause)
+![CI](https://img.shields.io/github/actions/workflow/status/SainsburyWellcomeCentre/WAZP/test_and_deploy.yml?label=CI)
+[![docs](https://img.shields.io/website?down_color=red&down_message=down&label=docs&up_color=brightgreen&up_message=up&url=https%3A%2F%2Fsainsburywellcomecentre.github.io%2FWAZP%2F)](https://sainsburywellcomecentre.github.io/WAZP/)
+[![Imports: isort](https://img.shields.io/badge/%20imports-isort-%231674b1?style=flat&labelColor=ef8336)](https://pycqa.github.io/isort/)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/python/black)
+[![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
 
-Project name is subject to refinement.
+# WAZP 🐝
+**W**asp **A**nimal-tracking **Z**oo project with **P**ose estimation
+(name is subject to refinement)
 
+## Overview
 
-## How to contribute
-### Setup
-* Clone the repository and install the package in editable mode (including all `dev` dependencies):
+WAZP is a dashboard built with [Dash-Plotly](https://dash.plotly.com/) for analysing animal tracking data. It is designed to work with pose estimation output from [DeepLabCut](https://github.com/DeepLabCut/DeepLabCut).
 
-```bash
+The package is currently in early development 🏗️ and is not yet ready for use. Stay tuned ⌛
+
+## Installation
+
+It is recommended to install WAZP inside a [conda](https://docs.conda.io/en/latest/) environment.
+Once you have `conda` installed, the following commands will create and activate a suitable environment, named `wazp-env`.
+
+```sh
+conda create -n wazp-env -c conda-forge python=3 pytables
+conda activate wazp-env
+```
+
+Next clone the GitHub repository, navigate to the local repository folder, and install WAZP.
+
+```sh
 git clone https://github.com/SainsburyWellcomeCentre/WAZP
 cd WAZP
-pip install -e '.[dev]'
+pip install .
 ```
```

Comment:
I can't leave a fancy suggestion here because my idea straddles a deleted line. 
Can't you use the fancy `pip` install direct from GitHub? A one-liner.

Reply:
Of course putting it on PyPI is the plan. But I was thinking we were delaying this until we have a minimally functional end-to-end product? 
I'm personally fine with releasing it as is though. In that case, however, what should be the version (currently it's `v0.1` + some dev tags and commit hashes that `setuptools_scm` adds.

Reply:
I generally favour releasing on PyPI at this stage, so that the users don't get one set of installation instructions that then get replaced by others. Version number doesn't really matter as long as it's <1 and we stick to semver.

Reply:
I will try to make the first PyPI release, right after this PR is merged